src/surfaces/actionBar/actionBar.jsx

```diff
@@ -71,108 +57,124 @@ const propTypes = {
         'space-between',
         'space-evenly',
     ]),
+    /**
+     * Assigns styling to the ActionBar dependant on
+     * whether it is a child of the Page or Drawer component.
+     */
     moduleType: PropTypes.oneOf(['drawer', 'page']),
-    style: PropTypes.shape({}),
+    /**
+     * Event handler for consumer to control state outside
+     * of the ActionBar.
+     */
     toggleSmSearchVisibleClassName: PropTypes.func,
 };
 
 const defaultProps = {
-    children: undefined,
+    children: null,
     classes: null,
-    className: undefined,
+    className: null,
     columns: [],
-    id: undefined,
+    id: null,
     justifyContent: 'flex-start',
-    moduleType: undefined,
-    style: {},
+    moduleType: null,
     toggleSmSearchVisibleClassName: null,
 };
 
 const styles = (theme) => {
+    const actionsButtonDrawerOptionHeight = 55;
     const standardThirdTiming = 300;
```

Comment:
Should this be 333? 😁 
I kid ... if 300 works, it's all good.

Reply:
Probably, but I don't think I should change it because of it potentially messing up the animation.

Reply:
Nm, I fixed it.

Reply:
Looks like we're now using "standard" transition durations defined by Material UI eh?  Me likey.
Resolving.